Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM python:3.7 as base
+
+RUN mkdir /app 
+WORKDIR /app
+COPY *.toml /app/
+
+ENV APP_INSTALL=/app
+ENV POETRY_HOME=/poetry
+ENV PATH=${POETRY_HOME}/bin:${PATH}
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+# DEVELOPMENT
+FROM base as development
+ENV FLASK_APP=app \
+    FLASK_ENV=development
+
+RUN poetry config virtualenvs.create false && \
+    poetry install --no-dev    
+    
+CMD ["poetry", "run", "flask", "run", "--host=0.0.0.0"]
+
+# PRODUCTION
+FROM base as production
+
+COPY runGunicorn.sh /app
+COPY todoapp /app/todoapp
+
+RUN poetry config virtualenvs.create false && \
+    poetry install --no-dev
+
+RUN cd /app
+
+RUN chmod 755 runGunicorn.sh && \
+    mkdir -p /var/log/gunicorn
+
+ENV FLASK_APP=app \
+    FLASK_ENV=production
```

Comment:
FLASK_APP=app is wrong but fortunately gets overridden when you run the container with `--env-file`. 

You should remove it here.